src/app/src/util/hooks.js

```diff
@@ -142,3 +144,104 @@ export default function useUpdateLeafletMapImperatively(
 
     return mapRef;
 }
+
+export function useCheckboxManager() {
```

Comment:
Could you please use arrow functions whenever possible? I aim to leverage the latest features of JavaScript.

Reply:
Ok, I don't see any obstacles to this.

Reply:
Done.